lib/plugins/inventory.js

```diff
@@ -641,6 +602,81 @@ function inject (bot, { version, hideErrors }) {
     bot.emit(`setWindowItems:${window.id}`)
   })
 
+  async function fish () {
+    let bobberDestroyed, getBobber, onParticles
+
+    async function onFishingDone () {
```

Comment:
add an err parameter here and if you get an error, throw it back otherwise fish() will succeed even if it fails to get anything